keeper/state.go

```diff
@@ -42,6 +43,7 @@ func (k *Keeper) GetChain(ctx context.Context) (uint16, error) {
 }
 
 // GetGuardianSets is a helper function for retrieving all guardian sets from state.
+// NOTE: this function never returns an error. Should we change signature?
```

Comment:
I'm definitely in favor of changing the method signature if no errors are ever returned